### ISSUE TITLE
"raw" input forbids %

### ISSUE THREAD

**Issue by pzinn** [KaTeX/KaTeX#2248]
so I finally got around to checking out the new html* commands. I was pretty excited about it until I got stuck with the following problem. say one uses htmlStyle. many css commands involve the symbol "%". but this creates an error.
for example,
```
\htmlStyle{background-color:rgb(255, 99, 71);}{x^2+1}
```
will work nicely but the equivalent syntax
```
\htmlStyle{background-color:hsl(9, 100%, 64%);}{x^2+1}
```
will throw a
KaTeX parse error: Unexpected end of input in raw at position 12: \htmlStyle{b̲a̲c̲k̲g̲r̲o̲u̲n̲d̲-̲c̲o̲l̲o̲r̲:̲h̲s̲l̲(̲9̲,̲ ̲1̲0̲0̲%, 64%);}{x^2+1…

the problem doesn't arise in say "url" input so shouldn't % be allowed in "raw" as well?

**Comment by pzinn** [comment]
weird, I can't find these commands any longer in
https://katex.org/docs/supported.html
did I dream them?

**Comment by ronkok** [comment]
No, you did not dream those commands. They are in PR #2082, which has been merged into master but not yet released. It has been seven months since a KaTeX release.

**Comment by bonkboykz** [comment]
> No, you did not dream those commands. They are in PR #2082, which has been merged into master but not yet released. It has been seven months since a KaTeX release.

Are there any updates on the 0.12.0 release? Would love to be able to use those new commands :)

**Comment by bonkboykz** [comment]
@ylemkimon @mpolyak

**Comment by kevinbarabash** [comment]
@bonkboykz 0.12.0 has been released.  See https://github.com/KaTeX/KaTeX/releases/tag/v0.12.0 for details.
